api/src/util/hoursUtilities.ts

```diff
@@ -26,19 +29,22 @@ export class HoursUtilities {
     } else {
       // Get date informatio
       const date = new Date();
-      const checkInDate = date.getDate().toString();
-      const checkInTime = date.getTime().toString();
 
       // Update user
       newUser.isCheckedIn = true;
-      newUser.transactions.push({
-        date: checkInDate,
-        checkIn: date.toString(),
-        checkOut: null,
-        checkInTime: checkInTime,
-        checkOutTime: null,
-        hours: null,
-      });
+      if (newUser.transactions) {
```

Comment:
Is this required? all users should have an empty transactions array from when they are created